editorial_board.md

```diff
@@ -0,0 +1,17 @@
+# Policies About the Editorial Boards
+
+These policies are currently a draft, and not yet in force.
+
+## Policies on Decisions
+Issues regarding governance structure, editorial board membership, leadership, and financial
+model must be decided by a 2/3 majority vote of the voting members of the editorial
```

Comment:
I think "of those members of the editorial board who vote on the issue" may be better than "of the voting members of the editorial board", as the latter leaves unclear whether we mean "those members of the editorial board who are entitled to vote" or "those members of the editorial board who actually voted". i realize this is probably my wording though so I'm probably editing myself. :)

Reply:
done